ingestion/src/metadata/ingestion/bulksink/metadata_usage.py

```diff
@@ -259,9 +274,9 @@ def __get_table_entity(
             entity_type=Table,
             service_name=self.service_name,
             filters={
-                "database": database_name,
+                "database": database,
```

Comment:
There's no need to go to ES, fqn.build already looks for ES data if we miss either the db or schema arguments